src/components/Resources/Quiz/Categories/QuizPage.jsx

```diff
@@ -150,30 +168,79 @@ export default function QuizPage() {
+                                                jnrjfnerjnjnijnegene vnv e vri jner v jnf erfref v feij ionz ezf &jv vi
+                                                invenvvjev vnvevev iefenrbvnjre e r en n iuznfjnerzfv nn n v{" "}
+                                                {questions[currentQuestion].additionalInfo}
+                                            </div>
+                                        </PopUpWindow>
+                                    ) : (
+                                        ""
+                                    )}
+                                    <div
+                                        style={{
+                                            display: "flex",
+                                            flexDirection: "row",
+                                            alignSelf: "end",
+                                            margin: "10px",
+                                            maxWidth: "100%",
+                                        }}
+                                    >
+                                        <SkipButton onClick={() => handleSkipButton(questions.length)}>
+                                            nextQuestion {`>>>`}
+                                        </SkipButton>
+                                    </div>
+                                </div>
                             ) : (
                                 ""
                             )}
                         </QuestionSection>
                         <AnswerSection>
                             {questions[currentQuestion].answerOptions.map((answerOption, i) => (
-                                <QuestionButton
-                                    key={i}
-                                    onClick={() => handleAnswerButtonClick(answerOption.isCorrect, questions.length, i)}
-                                    style={{
-                                        backgroundColor:
-                                            showAnswer && !answerOption.isCorrect && i === clickedAnswerIndex
-                                                ? "red"
-                                                : showAnswer && answerOption.isCorrect
-                                                ? "green"
-                                                : "",
-                                    }}
-                                    disabled={buttonClicked}
-                                >
-                                    {answerOption.answerText}
-                                </QuestionButton>
+                                <div key={i} style={{ position: "relative", zIndex: 0 }}>
+                                    {showAnswer && !answerOption.isCorrect && i === clickedAnswerIndex && (
+                                        <InfoButton onClick={(event) => handleAdditionalInfoButton(event)}>
+                                            <BsInfoCircle />
```

Comment:
on hover add tooltip - few words near to the icon, so the user will know on what they click, and that they should click it, or change to different icon which will indicate it better.